base/client.jl

```diff
@@ -298,11 +298,6 @@ function exec_options(opts)
     # load ~/.julia/config/startup.jl file
     startup && load_julia_startup()
 
-    if repl || is_interactive
```

Comment:
Unrelated?

Reply:
Good catch. I assume this was broken at some point, so carelessly deleted.